spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/AbstractLocalDeployerSupport.java

```diff
@@ -56,6 +59,12 @@
  */
 public abstract class AbstractLocalDeployerSupport {
 
+	protected static Set<Integer> usedPorts = Collections.newSetFromMap(new LinkedHashMap<Integer, Boolean>(){
+		protected boolean removeEldestEntry(Map.Entry<Integer, Boolean> eldest) {
```

Comment:
Would like to see `@Override` here

Reply:
:taco: and :+1:   (was looking for `:thumbsup` and found taco